toolkit.nu

```diff
@@ -107,7 +107,7 @@ export def test [
 export def "test stdlib" [
     --extra-args: string = ''
 ] {
-    cargo run -- -c $"
+    cargo run -- --no-config-file -c $"
```

Comment:
Maybe `-n` is good enough?

Reply:
Sure, I was just thinking it might be nice to write it out completely in places where it isn't repetitive